aldryn_newsblog/tests/frontend/integration/specs/spec.newsblog.crud.js

```diff
@@ -26,12 +26,144 @@ describe('Aldryn Newsblog tests: ', function () {
             }
 
             // wait for username input to appear
-            browser.wait(browser.isElementPresent(newsBlogPage.usernameInput),
-                newsBlogPage.mainElementsWaitTime);
+            browser.wait(function () {
```

Comment:
apparently promise based thing works not really the same way as a function based thing and sometimes fails for no reason